src/pages/api/registration.ts

```diff
@@ -3,10 +3,14 @@ import type { NextApiRequest, NextApiResponse } from 'next';
 import { Registration, validateRegistrationBody } from 'src/lib/registration';
 import { uploadToAirtable } from 'src/lib/airtable';
 
-export default async (req: NextApiRequest, res: NextApiResponse) => {
+export default async function register(
```

Comment:
@mcolella14 FYI lint complained about `anonymous export default` so I gave the function this name for now...let me know if you think there's a more suitable name for it.

Reply:
Ah ok I actually did the same thing [in this PR](https://github.com/mongodb/educators/pull/8/files#diff-3da78f93eaafcad98173138596688a4e15fbaf40e7f0e00ba7c4ecd18d221c71). So I'm fine either way!

Reply:
@mcolella14 oop missed that, let's go with yours and ill rebase once its merged